src/routes/console/wizard/cloudOrganizationChangeTier/choosePlan.svelte

```diff
@@ -0,0 +1,194 @@
+
+    $: if ($changeOrganizationTier.billingPlan === 'tier-0' && $changeTierSteps) {
+        $changeTierSteps = updateStepStatus($changeTierSteps, 2, true);
+        $changeTierSteps = updateStepStatus($changeTierSteps, 3, true);
+        $changeTierSteps = updateStepStatus($changeTierSteps, 4, true);
+    }
+
+    $: if (
+        $changeOrganizationTier.billingPlan === 'tier-2' ||
+        $changeOrganizationTier.billingPlan === 'tier-1'
+    ) {
+        $changeTierSteps = updateStepStatus($changeTierSteps, 2, false);
+        $changeTierSteps = updateStepStatus($changeTierSteps, 3, false);
+        $changeTierSteps = updateStepStatus($changeTierSteps, 4, false);
+    }
+
+    $: if ($changeOrganizationTier.billingPlan) {
+        $isUpgrade = $changeOrganizationTier.billingPlan > $organization.billingPlan;
+        checkOverUsage();
+    }
+
+    function checkOverUsage() {
+        if (!usage) return;
+        const plan = $plansInfo.plans.find(
+            (plan) => plan.$id === $changeOrganizationTier.billingPlan
+        );
+
+        const totBandwidth = usage?.bandwidth?.length > 0 ? usage.bandwidth[0].value : 0;
+        const totUsers = usage?.users?.length > 0 ? usage.users[0].value : 0;
+        $changeOrganizationTier.limitOverflow = {
+            bandwidth: totBandwidth > plan.bandwidth ? totBandwidth - plan.bandwidth : 0,
+            storage:
+                usage.storage[0] > sizeToBytes(plan.storage, 'GB')
+                    ? usage.storage[0] - plan.storage
+                    : 0,
+            users: totUsers > plan.users ? totUsers - plan.users : 0,
+            executions:
+                usage.executions[0] > plan.executions ? usage.executions[0] - plan.executions : 0,
+            members: members.total > plan.members ? members.total - (plan.members || Infinity) : 0
+        };
+        if (
+            $changeOrganizationTier.limitOverflow.bandwidth > 0 ||
+            $changeOrganizationTier.limitOverflow.storage > 0 ||
+            $changeOrganizationTier.limitOverflow.users > 0 ||
+            $changeOrganizationTier.limitOverflow.executions > 0 ||
+            $changeOrganizationTier.limitOverflow.members > 0
+        ) {
+            $changeOrganizationTier.isOverLimit = true;
+            $changeTierSteps = updateStepStatus($changeTierSteps, 4, false);
+        } else {
+            $changeOrganizationTier.isOverLimit = false;
+            $changeTierSteps = updateStepStatus($changeTierSteps, 4, true);
+        }
+
+        console.log($changeOrganizationTier.limitOverflow);
+    }
+
+    onMount(async () => {
+        usage = await sdk.forConsole.billing.listUsage($organization.$id);
+        members = await sdk.forConsole.teams.listMemberships($organization.$id);
+
+        //Select closest tier from starting one
+        // if ($organization.billingPlan === 'tier-2') {
+        //     $changeOrganizationTier.billingPlan = 'tier-1';
+        // }
+        // else
```

Comment:
leftover?

Reply:
We need this code when Scale plan is released